infra/gravity/node_commands.go

```diff
@@ -587,6 +593,57 @@ func (g *gravity) RunInPlanet(ctx context.Context, cmd string, args ...string) (
 	return out, nil
 }
 
+// IsLeader returns true if node is leader, false o.w.
+func (g *gravity) IsLeader(ctx context.Context) bool {
+	// TODO: use planet cmd once planet cli is updated s.t. `leader view` does
+	// not require args.
+	// leaderIP, err := g.RunInPlanet(ctx, "planet", "leader", "view")
+	leaderIP, err := g.RunInPlanet(ctx, "etcdctl", "get", "gravity/local/leader")
```

Comment:
should be a constant

Reply:
Do you mean to do something like:

`etcdGetLeader := []string{"etcdctl", "get","gravity/local/leader"}`
`leaderIP, err := g.RunInPlanet(ctx, etcdGetLeader[0], etcdGetLeader[1:]...)`

or did you mean to just make the etcd key a constant?

Reply:
more like `const LeaderKey="gravity/local/leader"`